docs/projects/amt/reporting-standard/schemas/modelcard_schema.json

```diff
@@ -0,0 +1,375 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/MinBZK/ai-validatie/main/docs/projects/amt/reporting-standard/schemas",
+    "title": "Model card schema",
+    "description": "JSON schema for model card",
+    "type": "object",
+    "properties": {
+        "provenance": {
+            "type": "object",
+            "properties":{
+                "git_commit_hash":{
+                    "type": "string",
+                    "description": "Git commit hash of the commit which contains the transformation file used to create this card"
+                },
+                "timestamp":{
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "A timestamp of the date, time and timezone of generation of this Model Card in ISO 8601 format"
+                },
+                "uri": {
+                    "type": "string",
+                    "description": "URI to the tool that was used to perform the transformations"
+                },
+                "author":{
+                    "type": "string",
+                    "description": "Name of person that initiated the transformations"
+                }
+            }
+        },
+        "language": {
```

Comment:
We could make this an enum with values valid languages codes as per the specification: https://www.iso.org/iso-639-language-code

Reply:
Where can i find the list of language codes? It seems there are a lot of different possible codes, this will maybe make a really large enum